packages/opentelemetry-exporter-collector/README.md

```diff
@@ -134,6 +134,18 @@ For GRPC please check [npm-url-grpc]
 
 For PROTOBUF please check [npm-url-proto]
 
+## Configuration options as environment variables
+
+Instead of providing options to `CollectorMetricExporter` and `CollectorTraceExporter` explicitly, environment variables may be provided instead.
+
+```sh
+OTEL_EXPORTER_OTLP_ENDPOINT=https://localhost:4317
```

Comment:
Can you also add the option to have specific trace endpoint and metrics endpoint? Useful in the case the user has different backend providers for different signals

Reply:
Done. Can you verify when you get a chance?